### DISCUSSION TITLE
snapshots parameter at restart option

### DISCUSSION THREAD

**Discussion by antoni7654** [erigontech/erigon#5877] [Q&A]
Hi:

If I do a restart with snapshots=false at a bsc node, will it be ok if the node was running properly? The node synced well from the contens of a snapshot file but cannot access to it externaly with wss and I'm trying some changes like adding tls options. I've restarted and has started to download a >200 Gb snapshot file and is takings some hours to get ready again. I haven't found the exact meaning of the snapshots parameter either at the read.me or the erigon commands help. Thanks!

**Comment by AskAlexSharov** [comment]
Erigon ignore this parameter after first startup

**Comment by AskAlexSharov** [comment]
See —torrent.* parameters to increase download speed

**Reply by antoni7654** [reply]
thanks Alex
